deployment/terraform/firewall.tf

```diff
@@ -0,0 +1,155 @@
+#
+# Bastion security group resources
+#
+resource "aws_security_group_rule" "bastion_ssh_ingress" {
+  type        = "ingress"
+  from_port   = 22
+  to_port     = 22
+  protocol    = "tcp"
+  cidr_blocks = ["${var.external_access_cidr_block}"]
+
+  security_group_id = "${module.vpc.bastion_security_group_id}"
+}
+
+resource "aws_security_group_rule" "bastion_ssh_egress" {
```

Comment:
We don't have anywhere to SSH now, but I think this makes sense to keep with the expectation that Batch support will be coming soon.